bootstrap-salt.sh

```diff
@@ -4325,6 +4353,179 @@ install_freebsd_restart_daemons() {
 
 #######################################################################################################################
 #
+#   OpenBSD Install Functions
+#
+
+install_openbsd_deps() {
+  ping -c 2 -q mirrors.nycbug.org || return 1
+  echo "installpath = http://mirrors.nycbug.org/pub/OpenBSD/${OS_VERSION}/packages/${CPU_ARCH_L}/" >/etc/pkg.conf || return 1
```

Comment:
@jfindlay, sure, my reason was  nycbug is closest to me official US mirror   
can use any official openbsd http mirror from the [list](http://www.openbsd.org/ftp.html#http) 
for example [ftp5.openbsd.org](http://ftp5.usa.openbsd.org/pub/OpenBSD/)
or can add more mirrors into the list